.github/workflows/php.yml

```diff
@@ -0,0 +1,64 @@
+name: PHP Composer
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version:
+          - "8.0"
+          - "8.1"
+
+        include:
+          - os: ubuntu-latest
```

Comment:
I know with our docker scripts we try to stay away from using `latest`. Do we want to specify a version here?

Reply:
Good question. I modeled this after the PHPUnit repo, and this is what they were using.

I'm not sure what would update in a newer version of a Linux distro that would cause this to fail, so I'm not sure if we would want to fail tests on it or not.